lib/install.js

```diff
@@ -241,14 +242,14 @@ function install (gyp, argv, callback) {
 
       req.on('close', function () {
         if (extractCount === 0) {
-          cb(new Error('Connection closed while downloading tarball file'))
+          cb(new Error('Connection closed while downloading tarball file "%s"', tarballUrl))
```

Comment:
I'm pretty sure this isn't going to work, it's not `console.log()`, you'll have to append